### ISSUE TITLE
Deprecated warnings during npm install process

### ISSUE THREAD

**Issue by dkocich** [hotosm/tasking-manager#901]
There are some warnings in the console which should be fixed by the update of turf...
```
npm install
npm WARN deprecated turf-bearing@3.0.12: Turf packages are now namespaced: please use @turf/bearing instead
npm WARN deprecated turf-collect@3.0.12: Turf packages are now namespaced: please use @turf/collect instead
npm WARN deprecated turf-bbox-polygon@3.0.12: Turf packages are now namespaced: please use @turf/bbox-polygon instead
npm WARN deprecated turf-bezier@3.0.12: Turf packages are now namespaced: please use @turf/bezier instead
npm WARN deprecated turf-along@3.0.12: Turf packages are now namespaced: please use @turf/along instead
npm WARN deprecated turf-area@3.0.12: Turf packages are now namespaced: please use @turf/area instead
npm WARN deprecated turf-combine@3.0.12: Turf packages are now namespaced: please use @turf/combine instead
npm WARN deprecated turf-concave@3.0.12: Turf packages are now namespaced: please use @turf/concave instead
npm WARN deprecated turf-center@3.0.12: Turf packages are now namespaced: please use @turf/center instead
npm WARN deprecated turf-centroid@3.0.12: Turf packages are now namespaced: please use @turf/centroid instead
npm WARN deprecated turf-circle@3.0.12: Turf packages are now namespaced: please use @turf/circle instead
npm WARN deprecated turf-convex@3.0.12: Turf packages are now namespaced: please use @turf/convex instead
npm WARN deprecated turf-distance@3.0.12: Turf packages are now namespaced: please use @turf/distance instead
npm WARN deprecated turf-envelope@3.0.12: Turf packages are now namespaced: please use @turf/envelope instead
npm WARN deprecated turf-buffer@3.0.12: Turf packages are now namespaced: please use @turf/buffer instead
npm WARN deprecated turf-destination@3.0.12: Turf packages are now namespaced: please use @turf/destination instead
npm WARN deprecated turf-bbox@3.0.12: Turf packages are now namespaced: please use @turf/bbox instead
npm WARN deprecated turf-difference@3.0.12: Turf packages are now namespaced: please use @turf/difference instead
npm WARN deprecated turf-tesselate@3.0.12: Turf packages are now namespaced: please use @turf/tesselate instead
npm WARN deprecated turf-explode@3.0.12: Turf packages are now namespaced: please use @turf/explode instead
npm WARN deprecated turf-hex-grid@3.0.12: Turf packages are now namespaced: please use @turf/hex-grid instead
npm WARN deprecated turf-inside@3.0.12: Turf packages are now namespaced: please use @turf/inside instead
npm WARN deprecated turf-intersect@3.0.12: Turf packages are now namespaced: please use @turf/intersect instead
npm WARN deprecated turf-isolines@3.0.12: Turf packages are now namespaced: please use @turf/isolines instead
npm WARN deprecated turf-flip@3.0.12: Turf packages are now namespaced: please use @turf/flip instead
npm WARN deprecated turf-kinks@3.0.12: Turf packages are now namespaced: please use @turf/kinks instead
npm WARN deprecated turf-line-distance@3.0.12: Turf packages are now namespaced: please use @turf/line-distance instead
npm WARN deprecated turf-line-slice@3.0.12: Turf packages are now namespaced: please use @turf/line-slice instead
npm WARN deprecated turf-nearest@3.0.12: Turf packages are now namespaced: please use @turf/nearest instead
npm WARN deprecated turf-meta@3.0.12: Turf packages are now namespaced: please use @turf/meta instead
npm WARN deprecated turf-midpoint@3.0.12: Turf packages are now namespaced: please use @turf/midpoint instead
npm WARN deprecated turf-point-on-line@3.0.12: Turf packages are now namespaced: please use @turf/point-on-line instead
npm WARN deprecated turf-point-on-surface@3.0.12: Turf packages are now namespaced: please use @turf/point-on-surface instead
npm WARN deprecated turf-simplify@3.0.12: Turf packages are now namespaced: please use @turf/simplify instead
npm WARN deprecated turf-sample@3.0.12: Turf packages are now namespaced: please use @turf/sample instead
npm WARN deprecated turf-random@3.0.12: Turf packages are now namespaced: please use @turf/random instead
npm WARN deprecated turf-point-grid@3.0.12: Turf packages are now namespaced: please use @turf/point-grid instead
npm WARN deprecated turf-square-grid@3.0.12: Turf packages are now namespaced: please use @turf/square-grid instead
npm WARN deprecated turf-square@3.0.12: Turf packages are now namespaced: please use @turf/square instead
npm WARN deprecated turf-tag@3.0.12: Turf packages are now namespaced: please use @turf/tag instead
npm WARN deprecated turf-tin@3.0.12: Turf packages are now namespaced: please use @turf/tin instead
npm WARN deprecated turf-within@3.0.12: Turf packages are now namespaced: please use @turf/within instead
npm WARN deprecated turf-invariant@3.0.12: Turf packages are now namespaced: please use @turf/invariant instead
npm WARN deprecated turf-helpers@3.0.12: Turf packages are now namespaced: please use @turf/helpers instead
npm WARN deprecated geojson-area@0.2.1: This module is now under the @mapbox namespace: install @mapbox/geojson-area instead
npm WARN deprecated turf-triangle-grid@3.0.12: Turf packages are now namespaced: please use @turf/triangle-grid instead
npm WARN deprecated turf-union@3.0.12: Turf packages are now namespaced: please use @turf/union instead
npm WARN deprecated geojson-normalize@0.0.0: This module is now under the @mapbox namespace: install @mapbox/geojson-normalize instead
npm WARN deprecated turf-planepoint@3.0.12: Turf packages are now namespaced: please use @turf/planepoint instead
npm WARN hot-tasking-manager@3.0.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
```

**Comment by dkocich** [comment]
This issue still occurs and there are some additional warnings - current log is here:
```
C:\git\tasking-manager\client>npm install
npm WARN deprecated ol3-geocoder@2.5.0: WARNING: This project has been renamed to ol-geocoder. Install using ol-geocoder instead.
npm WARN deprecated turf-collect@3.0.12: Turf packages are now namespaced: please use @turf/collect instead
npm WARN deprecated turf-along@3.0.12: Turf packages are now namespaced: please use @turf/along instead
npm WARN deprecated turf-area@3.0.12: Turf packages are now namespaced: please use @turf/area instead
npm WARN deprecated turf-bbox-polygon@3.0.12: Turf packages are now namespaced: please use @turf/bbox-polygon instead
npm WARN deprecated turf-bearing@3.0.12: Turf packages are now namespaced: please use @turf/bearing instead
npm WARN deprecated turf-bezier@3.0.12: Turf packages are now namespaced: please use @turf/bezier instead
npm WARN deprecated turf-buffer@3.0.12: Turf packages are now namespaced: please use @turf/buffer instead
npm WARN deprecated turf-center@3.0.12: Turf packages are now namespaced: please use @turf/center instead
npm WARN deprecated turf-circle@3.0.12: Turf packages are now namespaced: please use @turf/circle instead
npm WARN deprecated turf-centroid@3.0.12: Turf packages are now namespaced: please use @turf/centroid instead
npm WARN deprecated turf-combine@3.0.12: Turf packages are now namespaced: please use @turf/combine instead
npm WARN deprecated turf-concave@3.0.12: Turf packages are now namespaced: please use @turf/concave instead
npm WARN deprecated turf-convex@3.0.12: Turf packages are now namespaced: please use @turf/convex instead
npm WARN deprecated turf-destination@3.0.12: Turf packages are now namespaced: please use @turf/destination instead
npm WARN deprecated turf-distance@3.0.12: Turf packages are now namespaced: please use @turf/distance instead
npm WARN deprecated turf-envelope@3.0.12: Turf packages are now namespaced: please use @turf/envelope instead
npm WARN deprecated turf-explode@3.0.12: Turf packages are now namespaced: please use @turf/explode instead
npm WARN deprecated turf-difference@3.0.12: Turf packages are now namespaced: please use @turf/difference instead
npm WARN deprecated turf-bbox@3.0.12: Turf packages are now namespaced: please use @turf/bbox instead
npm WARN deprecated turf-flip@3.0.12: Turf packages are now namespaced: please use @turf/flip instead
npm WARN deprecated turf-tesselate@3.0.12: Turf packages are now namespaced: please use @turf/tesselate instead
npm WARN deprecated turf-hex-grid@3.0.12: Turf packages are now namespaced: please use @turf/hex-grid instead
npm WARN deprecated turf-inside@3.0.12: Turf packages are now namespaced: please use @turf/inside instead
npm WARN deprecated turf-intersect@3.0.12: Turf packages are now namespaced: please use @turf/intersect instead
npm WARN deprecated turf-isolines@3.0.12: Turf packages are now namespaced: please use @turf/isolines instead
npm WARN deprecated turf-kinks@3.0.12: Turf packages are now namespaced: please use @turf/kinks instead
npm WARN deprecated turf-line-slice@3.0.12: Turf packages are now namespaced: please use @turf/line-slice instead
npm WARN deprecated turf-line-distance@3.0.12: Turf packages are now namespaced: please use @turf/line-distance instead
npm WARN deprecated turf-midpoint@3.0.12: Turf packages are now namespaced: please use @turf/midpoint instead
npm WARN deprecated turf-nearest@3.0.12: Turf packages are now namespaced: please use @turf/nearest instead
npm WARN deprecated turf-planepoint@3.0.12: Turf packages are now namespaced: please use @turf/planepoint instead
npm WARN deprecated turf-point-grid@3.0.12: Turf packages are now namespaced: please use @turf/point-grid instead
npm WARN deprecated turf-point-on-line@3.0.12: Turf packages are now namespaced: please use @turf/point-on-line instead
npm WARN deprecated turf-point-on-surface@3.0.12: Turf packages are now namespaced: please use @turf/point-on-surface instead
npm WARN deprecated turf-random@3.0.12: Turf packages are now namespaced: please use @turf/random instead
npm WARN deprecated turf-sample@3.0.12: Turf packages are now namespaced: please use @turf/sample instead
npm WARN deprecated turf-simplify@3.0.12: Turf packages are now namespaced: please use @turf/simplify instead
npm WARN deprecated turf-square@3.0.12: Turf packages are now namespaced: please use @turf/square instead
npm WARN deprecated turf-tag@3.0.12: Turf packages are now namespaced: please use @turf/tag instead
npm WARN deprecated turf-square-grid@3.0.12: Turf packages are now namespaced: please use @turf/square-grid instead
npm WARN deprecated turf-triangle-grid@3.0.12: Turf packages are now namespaced: please use @turf/triangle-grid instead
npm WARN deprecated turf-tin@3.0.12: Turf packages are now namespaced: please use @turf/tin instead
npm WARN deprecated turf-union@3.0.12: Turf packages are now namespaced: please use @turf/union instead
npm WARN deprecated turf-within@3.0.12: Turf packages are now namespaced: please use @turf/within instead
npm WARN deprecated turf-meta@3.0.12: Turf packages are now namespaced: please use @turf/meta instead
npm WARN deprecated turf-helpers@3.0.12: Turf packages are now namespaced: please use @turf/helpers instead
npm WARN deprecated vector-tile@1.2.1: This module has moved: please install @mapbox/vector-tile instead
npm WARN deprecated turf-invariant@3.0.12: Turf packages are now namespaced: please use @turf/invariant instead
npm WARN deprecated geojson-area@0.2.1: This module is now under the @mapbox namespace: install @mapbox/geojson-area instead
npm WARN deprecated geojson-normalize@0.0.0: This module is now under the @mapbox namespace: install @mapbox/geojson-normalize instead
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated wrench@1.5.9: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
npm WARN deprecated point-geometry@0.0.0: This module has moved: please install @mapbox/point-geometry instead
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.

```

**Comment by xamanu** [comment]
Some of these have been fixed by @willemarcel in #1448

**Comment by xamanu** [comment]
We are running on updated dependencies now. All good. Closing here.
